spec/models/address_spec.rb

```diff
@@ -26,7 +26,7 @@
 
       it 'does not save the record' do
         address = Address.create(postcode: 'AA1 2BB', address_line_1: 'line 1')
-        expect(address.errors.full_messages).to eq(['Point geocoder lookup failed'])
+        expect(address.errors.full_messages).to eq(['Point geocoder lookup from postcode failed'])
```

Comment:
This message is a little terse. How about `We were unable to find a point in the UK with that postcode` or something like that?